purchase_order_restriction/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module does the following:
+
+* Adds Restrict Access boolean field in purchase order form for Purchase Manager and Purchase Order Owner to mark that the record is restricted
+* Adds new record rule for Purchase Users to have access only into their own record that have Restrict Access value is True
+* Adds Restrict Access to RFQ and Purchase Orders Tree view for Purchase Manager to monitor which records is currently restricted on the list
```

Comment:
This explanation is too detailed and it does not add much value while it adversely adds to the maintenance cost. We should not be listing every single detail of what's included in the module.

What's important in this documentation are:

- to roughly describes what the module does, and
- to explain the reason/background why we need this module

Our audience are not really the end-users, but the people with knowledge of Odoo development, who would just grasp the implementation details by reading the code.

```suggestion
This module adds Ristrict Access boolean field in the purchase order model, to limit the
access to the purchase order to 1) the buyer and 2) purchase managers.

The module can be useful if there are orders whose conditions should not be visible to
wide range of buyers (e.g. contract rates of the temp staff).
```